package.json

```diff
@@ -56,6 +56,7 @@
     "node-html-to-image": "^3.3.0",
     "nodemon": "^2.0.19",
     "octokit": "^3.1.0",
+    "parse5": "^7.1.2",
```

Comment:
Is this necessary?

Reply:
you mean one line above `octokit`? I think technically it's not needed because it's imported by probot but usually it's not a good idea to import indirect dependencies

Reply:
I don't see parse5 being used in your code.

Reply:
If I go to `Files changed` tab, I don't see parse5 as being added, only octokit 
![image](https://github.com/ubiquity/ubiquibot/assets/139262667/f3ecca9c-75c3-4df3-b93b-5eb3de77c732)

Reply:
Now me too. Something is up with your git because last time you somehow nuked my work on the config without displaying the deletion on your pull request.